chef_master/source/audit_supported_configurations.rst

```diff
@@ -5,151 +5,215 @@ Supported Configurations in ``audit`` Cookbook
+Fetch from Compliance and Report to Automate via Chef Server
+-------------------------------------------------------------------
 .. code-block:: ruby
 
-   ['audit']['reporter'] = 'chef-server-automate'
+   ['audit']['reporter'] = 'chef-server'
    ['audit']['server'] = 'https://compliance-server.test/api'
    ['audit']['refresh_token' OR 'token'] = '..'
    ['audit']['owner'] = 'User/Org'
    #Set in chef-server.rb:
-   data_collector['root_url'] = 'https://automate-server.test/data-collector/v0/'
+   data_collector['root_url'] = 'https://chef-automate.test/data-collector/v0/'
 
-Report to Compliance via Chef Server
-----------------------------------------
+Fetch from Compliance and Report to Compliance via Chef Server
+-------------------------------------------------------------------------------
 .. code-block:: ruby
 
    ['audit']['reporter'] = 'chef-server-compliance'
    ['audit']['server'] = 'https://compliance-server.test/api'
    ['audit']['refresh_token' OR 'token'] = '..'
    ['audit']['owner'] = 'User/Org'
 
-Fetch From Automate or Compliance via Chef Server
+Fetch From Compliance via Chef Server
 ==================================================
-Report Directly to Automate
-----------------------------------------
+Fetch From Compliance via Chef Server and Report Directly to Automate
+------------------------------------------------------------------------
 .. code-block:: ruby
 
    ['audit']['reporter'] = 'chef-automate'
-   ['audit']['fetcher'] = 'chef-server-automate'
+   ['audit']['fetcher'] = 'chef-server'
    ['audit']['server'] = 'https://compliance-server.test/api'
    ['audit']['refresh_token' OR 'token'] = '..'
    ['audit']['owner'] = 'User/Org'
    #Set in client.rb:
-   data_collector['server_url'] = 'https://automate-server.test/data-collector/v0/'
+   data_collector['server_url'] = 'https://chef-automate.test/data-collector/v0/'
    data_collector['token'] = '..'
 
 .. note: Must have Compliance Integrated w/ Chef Server
 
-Report Directly to Compliance
-----------------------------------------
+Fetch From Compliance via Chef Server and Report Directly to Compliance
+--------------------------------------------------------------------------
 .. code-block:: ruby
 
    ['audit']['reporter'] = 'chef-compliance'
-   ['audit']['fetcher'] = 'chef-server-automate'
+   ['audit']['fetcher'] = 'chef-server'
    ['audit']['server'] = 'https://compliance-server.test/api'
    ['audit']['refresh_token' OR 'token'] = '..'
    ['audit']['owner'] = 'User/Org'
 
 .. note:: Must have Compliance Integrated w/ Chef Server
 
-Report to Compliance via Chef Server
-----------------------------------------
+Fetch From Compliance via Chef Server and Report to Compliance via Chef Server
+-------------------------------------------------------------------------------
 .. code-block:: ruby
 
    ['audit']['reporter'] = 'chef-server-compliance'
-   ['audit']['fetcher'] = 'chef-server-automate'
+   ['audit']['fetcher'] = 'chef-server'
```

Comment:
`chef-server` fetcher is deprecated, only `chef-server-compliance` or `chef-server-automate` are valid entries